### ISSUE TITLE
[$250] Search - After receiving expense from employee, text sent is not shown in search - sent section

### ISSUE THREAD

**Issue by IuliiaHerets** [Expensify/App#52826]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** v9.0.64-4
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:** Y
**Issue reported by:** Applause Internal Team

## Action Performed:
1. Go to https://staging.new.expensify.com/home
2. Go to workspace settings
3. Invite a member
4. Open workspace chat with that member
5. Send a message
6. Navigate to LHN - search - chat - sent
7. Note the message sent is shown
8. Login as employee
9. Open workspace chat
10. Submit an expense
11. Login as admin
12. Go to search - expense
13. Note expense is shown
14. Open the member's workspace chat
15. Send a message after expense received from employee
16. Go to search - chat - sent
17. Note message sent is not shown
18. Refresh the page
19. Note the message sent before expense received is shown briefly and disappeared

## Expected Result:
In member's workspace chat, after receiving expense from employee, text sent must be shown in search - sent section.

## Actual Result:
In member's workspace chat, after receiving expense from employee, text sent is not shown in search - sent section.

## Workaround:
Unknown

## Platforms:

- [x] Android: Standalone
- [x] Android: HybridApp
- [x] Android: mWeb Chrome
- [ ] iOS: Standalone
- [ ] iOS: HybridApp
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

https://github.com/user-attachments/assets/06985d79-d00b-4fa2-8688-0ed54a8a4798

  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~021859706740749305880</li>
        <li>Upwork Job ID: 1859706740749305880</li>
        <li>Last Price Increase: 2024-11-21</li>
    </ul>
</details>

<details><summary>Issue Owner</summary>Current Issue Owner: @alitoshmatov</details>

**Comment by jliexpensify** [comment]
@IuliiaHerets I haven't been able to repro this on 64-4 so I want to confirm the first few steps: do you need to add a new member to the workspace, or does this affect existing members?

I haven't seen this behaviour with existing members.

**Comment by IuliiaHerets** [comment]
@jliexpensify Tester adding a new member for reproducing the issue

**Comment by jliexpensify** [comment]
I believe this is a #quality issue as chat is not working reliably. @alitoshmatov my Hybrid app and Android Chrome aren't v64, but my Chrome is - if you have newer versions, are you able to verify on those platforms?

**Comment by jliexpensify** [comment]
Not overdue, waiting on some testing from @alitoshmatov and some proposals.

**Comment by alitoshmatov** [comment]
Reproduced it, I think this should be handled on backend side since we don't have any logic to show search data, they are completely coming from backend.

cc: @jliexpensify

**Comment by FitseTLT** [comment]
This is a FE issue and being handle [here](https://github.com/Expensify/App/issues/52193). We can close it 👍

**Comment by jliexpensify** [comment]
Oh awesome, thanks @FitseTLT - closing in favour of the other GH. Appreciate your help @alitoshmatov!
